apps/studio/components/interfaces/Home/ProjectList/ProjectCard.tsx

```diff
@@ -25,7 +25,7 @@ const ProjectCard = ({
   const { name, ref: projectRef } = project
   const desc = `${project.cloud_provider} | ${project.region}`
 
-  const isBranchingEnabled = project.preview_branch_refs.length > 0
+  const isBranchingEnabled = project.preview_branch_refs && project.preview_branch_refs.length > 0
```

Comment:
@joshenlim i suppose this only makes sense if default project is also counted in the preview_branch_refs array

Reply:
yeah you're right - the api route for default project probably doesnt have this param hence why it crashed here